pkg/cloudprovider/providers/oci/load_balancer_spec.go

```diff
@@ -138,6 +138,10 @@ const (
 
 	// ServiceAnnotationLoadBalancerType is a service annotation for specifying lb type
 	ServiceAnnotationLoadBalancerType = "oci.oraclecloud.com/load-balancer-type"
+
+	// ServiceAnnotationLoadBalancerNodeFilter is a service annotation to select specific nodes as your backend in the LB
+	// based on label selector.
+	ServiceAnnotationLoadBalancerNodeFilter = "oci.oraclecloud.com/node-label-selector"
```

Comment:
Please add the annotation to the list of LB annotations here - https://github.com/oracle/oci-cloud-controller-manager/blob/master/docs/load-balancer-annotations.md 
The description, default value and what values it can take.

Reply:
We will be adding the changes suggested by @mrunalpagnis with another document change PR that is in pipe. So, marking as resolved for now.